pandas/core/indexes/datetimes.py

```diff
@@ -1937,8 +1937,10 @@ def tz_convert(self, tz):
                      mapping={True: 'infer', False: 'raise'})
     def tz_localize(self, tz, ambiguous='raise', errors='raise'):
         """
-        Localize tz-naive DatetimeIndex to given time zone (using
-        pytz/dateutil), or remove timezone from tz-aware DatetimeIndex
+        Convert aware Datetime index to TimeZone naive Datetime index.
```

Comment:
"Datetime index" -> DatetimeIndex 
 "TimeZone" -> time zone

And it is the other way around: you now say "convert aware to naive", but it is "localize naive to aware"

Reply:
okey sure

Reply:
@jorisvandenbossche can you illustrate more "Can you add an example for the None behaviour as well? (removing the timezone)"